admin_manual/installation/system_requirements.rst

```diff
@@ -12,31 +12,33 @@ Nextcloud needs a minimum of 128MB RAM, and we recommend a minimum of 512MB.
 Recommended setup for running Nextcloud
 ---------------------------------------
 
-For best performance, stability, support, and full functionality we recommend:
+For best performance, stability, and functionality we recommend:
 
 * Red Hat Enterprise Linux 7 / Ubuntu 16.04 LTS
 * MySQL/MariaDB
 * PHP 7.0 +
 * Apache 2.4 with mod_php
 
-Supported platforms
--------------------
+For a Nextcloud installation with efficient and reliable scaling over ~100 users, see our `deployment recommendations <https://portal.nextcloud.com/article/nextcloud-deployment-recommendations-7.html>`
 
-* Server: Linux (Debian 7, SUSE Linux Enterprise Server 11 SP3 & 12, 
+Working platforms
+-----------------
+
+* Server: Linux (Debian 8/9, SUSE Linux Enterprise Server 11 SP3 & 12, 
   Red Hat Enterprise Linux/CentOS 6.5 and 7 (7 is 64-bit only), Ubuntu 14.04 LTS, 16.04 LTS)
 * Web server: Apache 2 (mod_php, php-fpm) or Nginx (php-fpm) 
 * Databases: MySQL/MariaDB 5.5+; PostgreSQL 9, 10; Oracle 11g (currently only possible 
-  if you `contact us <https://nextcloud.com/enterprise>` as part of a subscription)
+  as part of an `enterprise subscription <https://nextcloud.com/enterprise>`) 
 * PHP 5.6 + required
 * Hypervisors: Hyper-V, VMware ESX, Xen, KVM
```

Comment:
"Xen, or KVM" would make it clear all are not needed.
Stating the hardware requirements, like Xen (Intel VT-x or AMD-V) might be nice.